SA/bibliography.bib

```diff
@@ -117,4 +117,27 @@ @online{bolero
 	title = {Getting Started | Bolero: F# in WebAssembly},
 	url = {https://fsbolero.io/docs/},
 	urldate = {2023-11-06}
-}
\ No newline at end of file
+}
+
+@online{spj-ghc-inference,
+   author = {Simon Peyton Jones},
+   title = {Type inference as constraint solving},
+   url = {https://simon.peytonjones.org/assets/ppts/type-inference.pptx},
+   date = {2019-06-01},
+   urldate = {2023-12-05}
+}
+
+@misc{mehta-unification,
+   author = {Prof. Dr. Farhad D. Mehta},
+   title = {PROGRAMMING IN PROLOG UNIFICATION AND PROOF SEARCH},
```

Comment:
```suggestion
   title = {Lecture slides: Programming in prolog unification and proof search},
```
I think we also need the year of the lecture to cite this correctly. I didn't see a date in the slides, so we'll pobably ask Farhad.

Reply:
Corrected title, and added a TODO